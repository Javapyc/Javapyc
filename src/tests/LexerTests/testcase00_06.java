// Bad characters in a comment: &$%@
*()-+=
