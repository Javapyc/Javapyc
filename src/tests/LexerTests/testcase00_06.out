Operator, *
Delimiter, (
Delimiter, )
Operator, -
Operator, +
Delimiter, =
